pypeclub/OpenPype
# Publisher: Easy access to publish page from create page

## Brief description
Added button to go from create page to publish page.

## Description
Added button over instance view which simplify change of page from create page to publish page. The button shows on hover of instance view (in half width of the widget).

## Additional info
Color changes or timing changes can be done. Colors are now hardcoded.

### Screenshot
![image](https://user-images.githubusercontent.com/43494761/202145334-75cd4233-e7e9-40fb-9cd0-46d8e7dcfc25.png)


## Testing notes:
1. Open Publisher (e.g. in tray publisher)
2. Hover over right side of the window on create page
3. A button should show
4. Click should get you to publish page
